ExtraEnemyCustomization/CustomAbilities/DrainStamina/DrainStaminaManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿using EECustom.Attributes;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EECustom.CustomAbilities.DrainStamina
+{
+    [CallConstructorOnLoad]
```

Comment:
What are you using `CallConstructorOnLoad` for?

Reply:
Networking reason, to ensure every player that launched game with plugin receive packets (without called at least once from some manager class or customizations)